docs/content/kv_commands.md

```diff
@@ -67,12 +69,21 @@ $ wrangler kv:namespace list
 
 ### `put`
 
-Writes a single key/value pair to the given namespace.
+Writes a single key/value pair to the given namespace. Optional params include 
+1. `--ttl`: Number of seconds for which the entries should be visible before they expire. At least 60. Takes precedence over 'expiration' option.
+2. `--expiration`: Number of seconds since the UNIX epoch, indicating when the key-value pair should expire.
+3. `--path`: Read value from the file at a given path. *This is good for security-sensitive operations, like uploading keys to KV; uploading from a file prevents a key value from being saved in areas like your terminal history.*
```

Comment:
```suggestion
1. `--path`: Read value from the file at a given path. *This is good for security-sensitive operations, like uploading keys to KV; uploading from a file prevents a key value from being saved in areas like your terminal history.*
```

Reply:
Markdown will renum them correctly, it's easier for adding/removing one later. Feel free to refuse if it seems overkilled to you.